src/popup/utils/utils.ts

```diff
@@ -71,6 +76,11 @@ export function includes<T, U extends T>(arr: readonly U[], elem: T): elem is U
   return arr.includes(elem as any);
 }
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
check recent PR's - Bart solved this issue

Reply:
Just rebase to develop.